From b3dbcdc372afda73291dc3b52bfcdb4b97a2a7b6 Mon Sep 17 00:00:00 2001
From: Andreas Gruenbacher <agruenba@redhat.com>
Date: Tue, 3 Jul 2018 10:31:32 +0200
Subject: [PATCH 06/12] attr_list.3: Fix the attributes.h include path

The header file is located at <attr/attributes.h>.  Make the attr_list.3
man page consistent with the other man pages.
---
 man/man3/attr_list.3 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/man3/attr_list.3 b/man/man3/attr_list.3
index c5ec40e..42ab92d 100644
--- a/man/man3/attr_list.3
+++ b/man/man3/attr_list.3
@@ -27,7 +27,7 @@ attr_list, attr_listf \- list the names of the user attributes of a filesystem o
 .PP
 .sp
 .nf
-.B #include <sys/attributes.h>
+.B #include <attr/attributes.h>
 .sp
 .B "int attr_list (const char \(**path, char \(**buffer, "
 .B "               const int buffersize, int flags,"
-- 
2.25.0

